app/components/support_interface/interview_details_component.rb

```diff
@@ -0,0 +1,38 @@
+module SupportInterface
+  class InterviewDetailsComponent < ViewComponent::Base
+    include ViewHelper
+
+    attr_reader :interview
+
+    def initialize(interview:)
+      @interview = interview
+    end
+
+    def rows
+      [
+        {
+          key: 'Provider',
+          value: interview.provider.name,
+        },
+        {
+          key: 'Meeting details',
```

Comment:
Interviews have both a `location` and an `additional_information` field so using the title Meeting details to render the location can cause confusion. 

Both of these text fields could also contain new lines, paragraph breaks and links that should be preserved and  linked to as well, so we should  use the markdown renderer to display their content.

Reply:
Good point, 'Meeting details' could be confusing. I think I'd prefer removing it altogether, as, from a support perspective, the key information is whether interviews have been scheduled for this application or not, whether some of them are cancelled etc. I'd go even further and say the support operator shouldn't have access to meeting details, e.g. zoom links, because links could be clicked and we definitely don't want that.

I tried to gather some requirements for this work and the key questions are: a. have interviews been scheduled, and b. do provider users choose to discard existing interviews and create new ones rather than modify existing ones. With 'Status', 'Created At' and 'Updated At' we should be able to tell.